### PR TITLE
fmt: reformat sound.c to noet and unify to vim code base

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
donot understand why was changing like that except messy.
// correct changes in https://github.com/vim/vim/pull/15969
// ref: https://github.com/vim/vim/discussions/13087#discussioncomment-7087803

### CODE REVIEW

**Comment by Shane-XB-Qian** [comment]
i think it should be more wise if to keep vim code base unified style, whatever you preferred fmt tool or sytle out of vim.
// so far only noticed sound.c and sign.c was changed, let me know if required to refmt others.

File: src/sound.c [hunk 1]
```diff
@@ -309,71 +309,71 @@ sound_free(void)
...
     {
-        LPCSTR clazz = "VimSound";
-        WNDCLASS wndclass = { 0, sound_wndproc, 0,    0, g_hinst, NULL, 0,
-                              0, NULL,          clazz };
-        RegisterClass(&wndclass);
-        g_hWndSound = CreateWindow(clazz, NULL, 0, 0, 0, 0, 0, HWND_MESSAGE,
-                                   NULL, g_hinst, NULL);
+	LPCSTR clazz = "VimSound";
+	WNDCLASS wndclass = { 0, sound_wndproc, 0,    0, g_hinst, NULL, 0,
+	    0, NULL,	      clazz };
```
Comment (glepnir) [inline src/sound.c]:
```suggestion
	WNDCLASS wndclass = { 0, sound_wndproc, 0,    0, g_hinst, NULL, 0,
	    0, NULL, clazz };
```

Comment (Shane-XB-Qian) [reply src/sound.c]:
```diff
-        WNDCLASS wndclass = {
-            0, sound_wndproc, 0, 0, g_hinst, NULL, 0, 0, NULL, clazz };
+        WNDCLASS wndclass = { 0, sound_wndproc, 0,    0, g_hinst, NULL, 0,
+                              0, NULL,          clazz };
```
last chg #16019 changed that, donot understand why given spaces there;
thx notice.

**Comment by chrisbra** [comment]
I just used `clang-format sound.c` and it produced exactly the current output. Same for sign.c. So let's keep it like this please.

**Comment by Shane-XB-Qian** [comment]
you maybe ok to use another tool to fmt,
but why you not insist to keep the style?
that's not a problem?
or what's the benefit get from that?
@yegappan and others admins seems not fully agreed these? though you may have said listen their opinion but looked their option were never respected. aha~ :smile:

--
shane.xb.qian

**Comment by chrisbra** [comment]
The reason is, it produces a consistent style and helps new contributors. There has been a bit of discussion about it, I suggest to revisit it.

**Comment by Shane-XB-Qian** [comment]
the only difference was clang_format etc such things maybe more common with others editors, compared to vim's native formater/method was for vim.
but please don't forget vim is an editor too, and it had its style already, it had consistent style already, it is not a new startup project.
just like vim's native syntax highlight, vs others, which also always some people popup asking if switch to use these etc.
I've read such "discussion" a lot, vim's style was not such bad, or actually much advance than most of them. you perhaps should insist the vim's style as more real "gatekeeper" instead of me! aha~ :smile:

--
shane.xb.qian

**Pull request closed without merging**